catalog/imaging/imaging-transformer-nitf/src/test/java/org/codice/alliance/transformer/nitf/common/CsexraAttributeTest.java

```diff
@@ -229,12 +229,840 @@ public void testSnowDepthMaxCategoryOther() throws NitfFormatException {
     @SuppressWarnings("unchecked")
     @Test
     public void testSnowDepthMaxCategoryNotSet() throws NitfFormatException {
-        when(tre.getFieldValue(CsexraAttribute.SNOW_DEPTH_CAT)).thenThrow(NitfFormatException.class);
+        when(tre.getFieldValue(CsexraAttribute.SNOW_DEPTH_CAT_NAME)).thenThrow(NitfFormatException.class);
 
         Serializable actual = CsexraAttribute.SNOW_DEPTH_MAX.getAccessorFunction()
                 .apply(tre);
 
         assertThat(actual, nullValue());
     }
 
+    @Test
+    public void testSensorPan() throws NitfFormatException {
+        when(tre.getFieldValue(CsexraAttribute.SENSOR_NAME)).thenReturn("PAN");
+
+        Serializable actual = CsexraAttribute.SENSOR.getAccessorFunction()
+                .apply(tre);
+
+        assertThat(actual, is(instanceOf(String.class)));
```

Comment:
Not sure we need to check that the instance if of type `String`. The next assert should fail if it isn't or if `actual` is `null`.